python/pipeline/utils/h5.py

```diff
@@ -94,6 +94,106 @@ def read_behavior_file(filename):
 
     return data
 
+def read_patchmaster(F):
+    if type(F)==str:
+        F=F[:-4]+'%d.h5' # chanage oneCell10_01.h5 to oneCell10_%d.h5
+        # Append '%d.h5' to filename in place of trailing 'x.h5'
+        # right now this way will only work for one digit file names, are we doing two digit names in the future? if so i suggest use the 0d format instead of having 9 and suddently 10 for eg.
+    else:
+        raise Exception ('currently the input format as {} is not surrpoted, please use str'.format(type(F)))
+    f=h5py.File(F,"r+",driver='family',memb_size=0) # read only, mimic if we are reading a batch of files, but the batch only have 0 member size
+    #begin case 7
+    '''
+        %% Files recorded after 03-28-2014 using the NPI ELC-03XS amplifier as amp 1 and the AxoClamp 2B (.1x headstage) as amp 2
+        %% Amp 2 current low-pass is set to 3000Hz.
+        %% Skips settings telegraph from NPI if any(~sets), i.e. if NPI amp is turned off.
+        %% Adds scanimage sync channel
+        %% Ball data is from optical encoder
+        %% ts2sec now takes 'packetLen' argument in order to correctly assign timestamps to end of data packets. This version adds a analogPacketLen field to data struct
+        % *** Packet length is set at 2000 for analog channels ***
+        '''
+    ANALOG_PACKET_LEN=2000
+    databall=f['ball']
+    wf=f['waveform']
+    sets=f['settings']
+    cam1ts=f['behaviorvideotimestamp']
+    cam2ts=f['eyetrackingvideotimestamp']
+    waveformDescStr=f.attrs['waveform Channels Description']
+    standardstr=b'Current Input 1, Voltage Input 1, Sync Photodiode, Stimulation Photodiode, LED Level Input, Patch Command Input, Shutter, Current Input 2, Voltage Input 2, Scan Image Sync'
+    if waveformDescStr!=standardstr:
+        raise Exception ('waveform Channels Description is wrong for this file version')
+    data={}
+    data['i1']=wf[0,:]# the wf is of shape 11, 6160000
+    data['v1']=wf[1,:]
+    data['i2']=wf[7,:]
+    data['v2']=wf[8,:]
+    data['syncPd']=wf[2,:]
+    data['stimPd']=wf[3,:]
+    data['led']=wf[4,:]
+    data['command']=wf[5,:]
+    data['shutter']=wf[6,:]
+    data['scanImage']=wf[9,:]
+    data['ts']=wf[10,:]
```

Comment:
can you run the formatting tool in your code editor. in pycharm, it is Ctrl+Alt+L. That will add empty lines when necessary, add spaces before and after the =, spaces before the commas, break lines that are too long, and other stuff like that